rising/utils/affine.py

```diff
@@ -106,28 +109,6 @@ def points_to_cartesian(batch: torch.Tensor) -> torch.Tensor:
     return batch[..., :-1] / batch[..., -1, None]
 
 
-def matrix_revert_coordinate_order(batch: torch.Tensor) -> torch.Tensor:
```

Comment:
We will still need this, if we want to apply it to points.

Because the PyTorch grid sample and affine grid expects the matrix to be in xy(z) and the tensor to transform in BxCx(Dx)HxW. If we now want to apply it to points, we should also use the (z)yx notation to be consistent with the images. We can therefore choose, to internally convert the points (and convert them back afterwards) or to convert the matrix with this (which is, what I actually prefer)

Reply:
Will be handled as discussed in our telco :)